x-pack/plugins/spaces/public/copy_saved_objects_to_space/components/space_result_details.tsx

```diff
@@ -123,7 +123,7 @@ export const SpaceCopyResultDetails = (props: Props) => {
           },
         };
         const selectContainerClass =
-          selectProps.options.length > 0 && isOverwritePending ? '' : ' euiAccordion-isClosed';
+          selectProps.options.length > 0 && isOverwritePending ? ' selectControl-isOpen' : '';
```

Comment:
You'll still need the full custom className here

```suggestion
          selectProps.options.length > 0 && isOverwritePending ? ' spcCopyToSpaceResultDetails__selectControl-isOpen' : '';
```

Reply:
I'm not sure that I follow. It's functioning as intended. They're two separate classes.

Open:

![image](https://user-images.githubusercontent.com/5295965/90187295-8d598880-dd87-11ea-95d7-a559075878d2.png)

Closed:
![image](https://user-images.githubusercontent.com/5295965/90187359-a3674900-dd87-11ea-9796-8409a558f932.png)

The EuiAccordion functions the same, when it's open the div has the `euiAccordion` and `euiAccordion-isOpen` classes, but when it's closed it just has the `euiAccordion` class.